tune_exercises/exercise_1_basics.ipynb

```diff
@@ -200,8 +200,8 @@
     "\n",
     "results = tune.run(\n",
     "    tune_iris, \n",
-    "    config={\"lr\": 0.1, \"dense_1\": 1, \"dense_2\": 0.1},\n",
-    "    num_samples=1,\n",
+    "    config={\"lr\": 0.1, \"dense_1\": 1, \"dense_2\": 2},\n",
+    "    num_samples=10,\n",
```

Comment:
```suggestion
    "    num_samples=2,\n",
```

Reply:
Should we likewise change the assertion below to 2 as well?